src/components/Uploadcare/lib/effects.ts

```diff
@@ -20,23 +20,27 @@ const resize = (width: number | null = null, height: number | null = null) => {
     }
 
     return `/resize/${safeWidth}x${safeHeight}/`;
-};
+}
 
-export const crop = (width: number, height: number) => {
+export function crop(width: number, height: number) {
     const safeWidth = Math.min(width, MAX_WIDTH);
     const safeHeight = Math.min(height, MAX_HEIGHT);
 
     return `/crop/${safeWidth}x${safeHeight}/`;
-};
+}
 
-export const format = (format: ImageExtension | VideoExtension | "auto") =>
-    `/format/${format}/`;
+export function progressive(value: boolean = true) {
+    return value ? `/progressive/yes/` : '';
+}
 
-export const grayscale = () => "/grayscale/";
+export function quality(quality: 'smart' = 'smart') {
```

Comment:
There are [other quality settings](https://uploadcare.com/docs/transformations/image/compression/#operation-quality) which we could use as well, I think it's worth it to reflect them in the types